### PR TITLE
ProfileComponent and EditProfileComponent

### PR DESCRIPTION
**Pull request by jazlynnfok**

### CODE REVIEW

File: src/readit/profile/profile-public.js
Comment (dennissmith617) [inline src/readit/profile/profile-public.js:102]: Should we remove commented out code?

**Review by dennissmith617 — APPROVED** [review]
small question on commented out code

File: src/readit/profile/profile-public.js
Comment (jazlynnfok) [reply src/readit/profile/profile-public.js:102]: I left this in as a reminder to add reviews/comments that should come from the Details page. I can wait until then to merge this PR.

**Pull request merged**
